src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static final String SEPARATE = System.lineSeparator();
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                int index = line.indexOf(",");
+                int length = line.substring(index).length();
+                int value = Integer.parseInt(line.substring(index + 1, index + length));
```

Comment:
Please, use split() instead of substring()
see [common mistakes ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#use-split-instead-of-substring)